book/wildfires/sensors/wildfires-sensors-modis.ipynb

```diff
@@ -0,0 +1,589 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ! wget 'https://firms2.modaps.eosdis.nasa.gov/data/country/zips/modis_2020_all_countries.zip' -O 'modis_2020_all_countries.zip'
</pre><p> </p><p>I would suggest to replace by <code>urllib.request.urlretrieve </code>as wget isn't installed by default in all OS.</p><p> </p><pre class="ql-syntax" spellcheck="false">filepath = 'https://firms2.modaps.eosdis.nasa.gov/data/country/zips/'
filename = 'modis_2020_all_countries.zip'
urllib.request.urlretrieve(filepath+filename, '../sensors/' + filename)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/acocac/environmental-ai-book/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>